src/main/java/io/aiven/kafka/connect/transforms/TombstoneHandlerConfig.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2019 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.transforms;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigException;
+
+public final class TombstoneHandlerConfig extends AbstractConfig {
+
+    public static final String TOMBSTONE_HANDLER_BEHAVIOR = "behavior";
+
+    public TombstoneHandlerConfig(final Map<?, ?> originals) {
+        super(config(), originals);
+    }
+
+    static ConfigDef config() {
+        return new ConfigDef()
+            .define(
+                TOMBSTONE_HANDLER_BEHAVIOR,
+                ConfigDef.Type.STRING,
+                Behavior.IGNORE.name(),
```

Comment:
I propose to not introduce any default value for this config (`ConfigDef.NO_DEFAULT_VALUE`). If a user adds this, let them define something explicitly.